src/Language/PureScript/Ide/Util.hs

```diff
@@ -152,4 +154,9 @@ ideReadFile = ideReadFile' readUTF8FileT
 -- rather than @TextMode@ line endings get mangled on Windows otherwise. This is
 -- irrelevant for parsing, because the Lexer strips these additional @\r@'s.
 ideReadTextFile :: (MonadIO m, MonadError IdeError m) => FilePath -> m Text
-ideReadTextFile = ideReadFile' TIO.readFile
+ideReadTextFile =
+  ideReadFile' (map TE.decodeUtf8 . readFileNonBinary)
+  where
+    readFileNonBinary fp = withFile fp ReadMode $ \h -> do
+      hSetBinaryMode h False
```

Comment:
Unfortunately that doesn't work. It seems hGetContents just ignores Binary/TextMode.